script.js

```diff
@@ -1,52 +1,70 @@
-const library = document.querySelector('.library');
-const titleInput = document.querySelector('#title');
-const authorInput = document.querySelector('#author');
-let removeBtns = document.querySelectorAll('.removeButton');
+import Book from './book.js';
 
-let books = JSON.parse(localStorage.getItem('books')) || [];
+class Library {
+  constructor() {
+    this.library = document.querySelector('.library');
+    this.titleInput = document.querySelector('#title');
+    this.authorInput = document.querySelector('#author');
```

Comment:
- [ ] Great job working on the refactoring but I think the functionality for adding and removing since when I try to add a new book I can't see it so please check the getBooks method maybe there is an issue there so please make it works as expected ⭐💪🏻🔥🤩





https://github.com/Gh-Reza/AwsomeBooks/assets/58553711/350c0148-cd3b-49ed-9638-1eed27d2bc0c

Reply:
Hello @Reem-lab, thank you for your effort. I don't know the problem, but it works properly on both local and GitHub pages. Can you kindly test it on github pages?

Reply:
Hello @Gh-Reza I am so sorry but I checked it many times but I can't add a book! I will check again